lib/wallet/integration/src/Test/Integration/Scenario/API/Byron/Migrations.hs

```diff
@@ -517,7 +527,7 @@ spec = describe "BYRON_MIGRATIONS" $ do
                 , expectErrorMessage errMsg400ParseError
                 ]
 
-    Hspec.it "BYRON_MIGRATE_08 - \
```

Comment:
Changing it from `Hspec.it` to `Hspec.Extra.it` will re-add retries. I suspect they were disabled for a reason, but I also don't per se see why so seems fine to me.